CHANGELOG.md

```diff
@@ -4,8 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## 1.0.1 - 2022-10-22
```

Comment:
Is this date right? It's a saturday is all.

Reply:
I'm a bit optimistic that it'll be merged by tomorrow to the latest :smile:

Reply:
and released...

Reply:
I just realized... you don't need to do this anymore so you could remove this entirely if you want? With the new version of beehive flow it will add this. Alternatively you can keep this and it'll just update the date when you actually release 😄